examples_test.go

```diff
@@ -267,13 +267,17 @@ func Example_race_selectors() {
 	page.MustElement("#id_password").MustInput(password).MustPress(input.Enter)
 
 	// It will keep retrying until one selector has found a match
-	page.Race().MustElement(".nav-user-icon-base", func(el *rod.Element) {
+	elm := page.Race().Element(".nav-user-icon-base").
+		Element("[data-cy=sign-in-error]").
+		MustDo()
+
+	if elm.MustMatches(".nav-user-icon-base") {
```

Comment:
Can we use the `MustHandle` here? so we can reduce an unnecessary global variable `elm` and repeating selectors. Also, it's better to demonstrate as many methods as possible.

Reply:
Yes, will have it done!